modules/porous_flow/src/base/PorousFlowApp.C

```diff
@@ -116,7 +118,8 @@ PorousFlowApp::registerObjects(Factory & factory)
   registerMaterial(PorousFlowMaterialPorosityConst);
   registerMaterial(PorousFlowPorosityHM);
   registerMaterial(PorousFlowMaterialMassFractionBuilder);
-  registerMaterial(PorousFlowMaterialPermeabilityConst);
+  registerMaterial(PorousFlowPermeabilityConst);
```

Comment:
Since you include PorousFlowPermeabilityUnity, you probably want to registerMaterial(...Unity) too